README.md

```diff
@@ -102,6 +102,7 @@
 * [Cachix](https://cachix.org/) - Hosted binary cache service; free for open-source projects.
 * [devshell](https://github.com/numtide/devshell) - `mkShell` with extra bits and a toml config option to be able to onboard non-nix users.
 * [dream2nix](https://github.com/nix-community/dream2nix) - A framework for automatically converting packages from other build systems to Nix.
+* [flake.parts](https://github.com/hercules-ci/flake-parts) - Minimal Nix modules for Flakes: split your flakes into modules and get things done with community modules.
```

Comment:
I like most of this description. Add the word "framework" after "Minimal Nix modules", and I think it'd be perfect.